plugins/item_tasks/web_client/views/SelectSingleFileTaskWidget.js

```diff
@@ -0,0 +1,68 @@
+import View from 'girder/views/View';
+
+import 'girder/utilities/jquery/girderEnable';
+import 'girder/utilities/jquery/girderModal';
+
+import PaginateTasksWidget from './PaginateTasksWidget';
+import ItemTaskCollection from '../collections/ItemTaskCollection';
+
+import SelectTaskViewDialogTemplate from '../templates/selectTaskViewDialog.pug';
```

Comment:
Renaming all these pug and styl files made me wish the file structure was simpler for front end dev. Something like each widget or view having its own folder, that way file names could be more relative and less absolute and wordy.